cocos2d/core/CCNode.js

```diff
@@ -52,8 +52,24 @@ var _mat4_temp = math.mat4.create();
 var _vec3_temp = math.vec3.create();
 var _quat_temp = math.quat.create();
 var _globalOrderOfArrival = 1;
-var _cachedArray = new Array(16);
-_cachedArray.length = 0;
+
+var _cachedArrayPool = new Array(3);
+_cachedArrayPool.length = 0;
+
+_cachedArrayPool.push(new Array(16));
+_cachedArrayPool.push(new Array(16));
+
```

Comment:
why need push 2 array to _cachedArrayPool？

Reply:
preallocate two cache array